benchmark.py

```diff
@@ -233,6 +266,19 @@ def RunBenchmark(args, logfile):
                          logfile=logfile, zone=args.zone, project=args.project)
   print "Data stored into database."
 
+  if args.create_delete:
+    print "Deleting instance. Wait..."
+    # pexpect.run does not take argument as arrays
+    pexpect.run(("gcloud compute --project {}"
+                 " instances delete {} --zone {}").format(
+                     args.project, args.vm_name, args.zone),
+                events={"Do you want to continue (Y/n)?": "Y\n"},
```

Comment:
BTW, you shouldn't need to interactively do this Y/N business with gcloud, it has a `--quiet` setting to allow batch execution. See `--quiet` in https://cloud.google.com/sdk/gcloud/reference/. Can you file an issue or a PR to clean this up?